Gemfile

```diff
@@ -16,10 +16,12 @@ gem 'sass-rails', '~> 5.0'
 gem 'turbolinks'
 gem 'uglifier', '>= 1.3.0'
 
-group :development do
-  gem 'pry-byebug'
-  gem 'spring'
-  gem 'web-console', '~> 2.0'
+group :development, :test do
+  gem 'rspec-rails'
+  gem 'shoulda-matchers'
+  gem 'factory_bot_rails'
+  gem 'capybara'
+  gem 'coveralls', require: false
```

Comment:
you have added the coveralls gem but are missing the configuration for coveralls